### PR TITLE
Update release version v0.37.0

### PR DESCRIPTION
**Pull request by vishwahiremat**
# Description

Update release version v0.37.0

## Type of change

- This pull request fixes a bug in Radius and has an approved issue (issue link required).
- This pull request adds or changes features of Radius and has an approved issue (issue link required).
- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

Fixes: #issue_number

### CODE REVIEW

File: docs/release-notes/v0.37.0.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+Today we're happy to announce the release of Radius v0.37.0. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
```
Comment (Reshrahim) [inline docs/release-notes/v0.37.0.md:12]:
```suggestion

### Radius AWS Cloud provider supports IAM Roles for Service Accounts (IRSA)
You can now configure your AWS cloud provider to use AWS federated identity (IRSA) to deploy and manage AWS resources. With this support, infrastructure operators are not burdened with the rotation of the credentials anymore. More information on [how to configure the AWS IRSA](https://docs.radapp.io/guides/operations/providers/aws-provider/howto-aws-provider-irsa/)

### Radius is merged with the official Bicep
Previously Radius used a fork of the Bicep to author and deploy Radius and AWS types. With 0.37 release we have merged our extensibility updates with the upstream and are deprecating the temporary fork of Bicep. To use Radius features with the official Bicep compiler, you need to create a `bicepconfig.json` with the extensions. More information [here](https://docs.radapp.io/guides/tooling/bicepconfig/overview/)

### Test Improvements 
We separated the cloud tests and non-cloud tests in our functional  test workflows to improve the test verification process for contributors. In the new workflow the non-cloud test workflows will trigger automatically when pull requests are submitted, while tests using cloud resources will need approval from approvers/maintainers to be triggered. We also fixed an issue with `magpiego` image failing for MacOS+M1 chip users by publishing a multi-architectural image. 

```

File: docs/release-notes/v0.37.0.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+## Breaking changes
+
+N/A
```
[the remark this answers is not in the document]
Comment (Reshrahim) [reply docs/release-notes/v0.37.0.md:15]: @sk593 and @vishwahiremat - Please review and validate the breaking changes

[the remark this answers is not in the document]
Comment (vishwahiremat) [reply docs/release-notes/v0.37.0.md:15]: looks good to me!

File: docs/release-notes/v0.37.0.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+### Test Improvements
+We separated the cloud tests and non-cloud tests in our functional  test workflows to improve the test verification process for contributors. In the new workflow the non-cloud test workflows will trigger automatically when pull requests are submitted, while tests using cloud resources will need approval from approvers/maintainers to be triggered. We also fixed an issue with `magpiego` image failing for MacOS+M1 chip users by publishing a multi-architectural image.
+
+## Breaking changes
+
+- Radius merge to official Bicep introduces some breaking changes. To use Radius features with the official Bicep compiler please do the following:
+    1. Create a bicepconfig.json with the extensions to author and deploy Radius and AWS types. More information here.
+    2. Update the import statements in the existing bicep files
+        - import radius as radius should become extension radius to use Radius types
```
Comment (Reshrahim) [inline docs/release-notes/v0.37.0.md:27]:
```suggestion
        - `import radius as radius` should become `extension radius` to use Radius types
```

File: docs/release-notes/v0.37.0.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        - import aws as aws should become extension aws to use AWS types
```
Comment (Reshrahim) [inline docs/release-notes/v0.37.0.md:28]:
```suggestion
        - `import aws as aws` should become `extension aws` to use AWS types
```

File: docs/release-notes/v0.37.0.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+        - import kubernetes as kubernetes {} should become extension kubernetes with {} as kubernetes to use Kubernetes types
```
Comment (Reshrahim) [inline docs/release-notes/v0.37.0.md:29]:
```suggestion
        - `import kubernetes as kubernetes {}` should become `extension kubernetes with {} as kubernetes to use Kubernetes types`
```

File: docs/release-notes/v0.37.0.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+    3. If you have Radius-Bicep extension installed, please disable and install the official bicep extension. More details here.
```
Comment (Reshrahim) [inline docs/release-notes/v0.37.0.md:30]:
```suggestion
    3. If you have Radius-Bicep extension installed, please disable and install the official bicep extension. More details [here](https://edge.docs.radapp.io/guides/tooling/vscode/howto-vscode-bicep/).
```

File: docs/release-notes/v0.37.0.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+### Radius AWS Cloud provider supports IAM Roles for Service Accounts (IRSA)
+You can now configure your AWS cloud provider to use AWS federated identity (IRSA) to deploy and manage AWS resources. With this support, infrastructure operators are not burdened with the rotation of the credentials anymore. More information on [how to configure the AWS IRSA](https://docs.radapp.io/guides/operations/providers/aws-provider/howto-aws-provider-irsa/)
+
+### Test Improvements
+We separated the cloud tests and non-cloud tests in our functional  test workflows to improve the test verification process for contributors. In the new workflow the non-cloud test workflows will trigger automatically when pull requests are submitted, while tests using cloud resources will need approval from approvers/maintainers to be triggered. We also fixed an issue with `magpiego` image failing for MacOS+M1 chip users by publishing a multi-architectural image.
+
+## Breaking changes
+
+- Radius merge to official Bicep introduces some breaking changes. To use Radius features with the official Bicep compiler please do the following:
```
Comment (Reshrahim) [inline docs/release-notes/v0.37.0.md:24]:
```suggestion
- **Radius merge to official Bicep** introduces some breaking changes. To use Radius features with the official Bicep compiler please do the following:
```

File: docs/release-notes/v0.37.0.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+    1. Create a bicepconfig.json with the extensions to author and deploy Radius and AWS types. More information here.
+    2. Update the import statements in the existing bicep files
+        - import radius as radius should become extension radius to use Radius types
+        - import aws as aws should become extension aws to use AWS types
+        - import kubernetes as kubernetes {} should become extension kubernetes with {} as kubernetes to use Kubernetes types
+    3. If you have Radius-Bicep extension installed, please disable and install the official bicep extension. More details here.
+
+- AWS IRSA support introduces some breaking changes to rad credential register aws CLI command. With the additional identity support, rad credential register aws is now rad credential register aws access-key to register the IAM access-key and rad credential register aws to register the AWS IAM Roles for Service Accounts(IRSA).
```
Comment (Reshrahim) [inline docs/release-notes/v0.37.0.md:32]:
```suggestion
- **AWS IRSA support** introduces some breaking changes to rad credential register aws CLI command. With the additional identity support, rad credential register aws is now [rad credential register aws access-key](https://docs.radapp.io/reference/cli/rad_credential_register_aws_access-key/) to register the IAM access-key and [rad credential register aws](https://docs.radapp.io/reference/cli/rad_credential_register_aws_irsa/) to register the AWS IAM Roles for Service Accounts(IRSA).
```

File: docs/release-notes/v0.37.0.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sk593) [reply docs/release-notes/v0.37.0.md]: lgtm as well

File: docs/release-notes/v0.37.0.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+### Radius AWS Cloud provider supports IAM Roles for Service Accounts (IRSA)
+You can now configure your AWS cloud provider to use AWS federated identity (IRSA) to deploy and manage AWS resources. With this support, infrastructure operators are not burdened with the rotation of the credentials anymore. More information on [how to configure the AWS IRSA](https://docs.radapp.io/guides/operations/providers/aws-provider/howto-aws-provider-irsa/)
+
+### Test Improvements
+We separated the cloud tests and non-cloud tests in our functional  test workflows to improve the test verification process for contributors. In the new workflow the non-cloud test workflows will trigger automatically when pull requests are submitted, while tests using cloud resources will need approval from approvers/maintainers to be triggered. We also fixed an issue with `magpiego` image failing for MacOS+M1 chip users by publishing a multi-architectural image.
+
+## Breaking changes
+
+**Radius merge to official Bicep** introduces some breaking changes. To use Radius features with the official Bicep compiler please do the following:
```
Comment (sk593) [inline docs/release-notes/v0.37.0.md:24]: Formatting looks a little off. I think newlines need to be added


Screenshot 2024-08-19 at 9 58 59 AM

Comment (vishwahiremat) [reply docs/release-notes/v0.37.0.md]: fixed it

**Review by lakshmimsft — APPROVED** [review]

**Pull request merged**